zkSNACKs/WalletWasabi
# [PoF] ColdCard Kata failure

CC won't sign the tx and will freeze while displaying `Validating...` It can be recovered only with a power cycle.  

## How to reproduce

1. Init CC with there Recovery words: `more maid moon upgrade layer alter marine screen benefit way cover alcohol`
2. Run the test called ColdCardKataAsync(), like `dotnet test --filter ColdCardKataAsync`
3. See error

## Screenshots

![image](https://user-images.githubusercontent.com/9844978/91740185-37704780-ebb3-11ea-9f84-2a0adf109020.png)

## Error logs

HWI producing this output with -debug option enabled
```
output:{"error": "read error", "code": -13}
error:Waiting for OK on the Coldcard...
    Traceback (most recent call last):
      File "hwilib\errors.py", line 104, in handle_errors
      File "hwilib\cli.py", line 274, in process_commands
      File "hwilib\cli.py", line 74, in signtx_handler
      File "hwilib\commands.py", line 85, in signtx
      File "hwilib\devices\coldcard.py", line 60, in func
      File "hwilib\devices\coldcard.py", line 161, in sign_tx
      File "hwilib\devices\ckcc\client.py", line 141, in send_recv
      File "hid.pyx", line 122, in hid.device.read
    OSError: read error
```

Related: https://github.com/zkSNACKs/WalletWasabi/pull/4251

Don't care about the CI tests - those are failing because of the debug code.
